src/contributors.py

```diff
@@ -0,0 +1,16 @@
+import json
+
+contributors_json = {}
+
+
+def get_contributors():
+  return contributors_json
```

Comment:
Oh, sorry we also need to declare the global property here too.

```suggestion
  global contributors_json
  return contributors_json
```